cs-jClouds/src/main/java/io/cloudslang/content/jclouds/actions/volumes/DeleteVolumeInRegionAction.java

```diff
@@ -0,0 +1,86 @@
+public class DeleteVolumeInRegionAction {
+    /**
+     * Deletes the specified EBS volume. The volume must be in the "available" state (not attached to an instance).
+     * <p>
+     * Note: The volume may remain in the deleting state for several minutes. For more information, see Deleting an Amazon
+     * EBS Volume in the Amazon Elastic Compute Cloud User Guide.
+     *
+     * @param provider   Cloud provider on which you have the instance - Valid values: "amazon" or "openstack".
+     * @param endpoint   Endpoint to which first request will be sent. Ex: "https://ec2.amazonaws.com" for amazon or
+     *                   "http://hostOrIp:5000/v2.0" for OpenStack.
+     * @param identity   Optional - Username of your account or the Access Key ID. For OpenStack provider the required
+     *                   format is 'alias:username'.
+     * @param credential Optional - Password of the user or the Secret Access Key that correspond to the identity input.
+     * @param proxyHost  Optional - Proxy server used to access the web site. If empty no proxy will be used.
+     * @param proxyPort  Optional - Proxy server port.
+     * @param region     Optional - region where volume to be deleted belongs. Ex: "RegionOne", "us-east-1".
+     *                   ListRegionAction can be used in order to get all regions - Default: "us-east-1"
+     * @param volumeId   ID of the EBS volume.
+     * @return A map with strings as keys and strings as values that contains: outcome of the action, returnCode of the
+     * operation, or failure message and the exception if there is one
+     */
+    @Action(name = "Delete Volume in Region",
+            outputs = {
+                    @Output(Outputs.RETURN_CODE),
+                    @Output(Outputs.RETURN_RESULT),
+                    @Output(Outputs.EXCEPTION)
+            },
+            responses = {
+                    @Response(text = Outputs.SUCCESS, field = Outputs.RETURN_CODE, value = Outputs.SUCCESS_RETURN_CODE,
+                            matchType = MatchType.COMPARE_EQUAL, responseType = ResponseType.RESOLVED),
+                    @Response(text = Outputs.FAILURE, field = Outputs.RETURN_CODE, value = Outputs.FAILURE_RETURN_CODE,
+                            matchType = MatchType.COMPARE_EQUAL, responseType = ResponseType.ERROR)
+            }
+    )
+    public Map<String, String> execute(@Param(value = Inputs.CommonInputs.PROVIDER, required = true) String provider,
+                                       @Param(value = Inputs.CommonInputs.ENDPOINT, required = true) String endpoint,
+                                       @Param(Inputs.CommonInputs.IDENTITY) String identity,
+                                       @Param(value = Inputs.CommonInputs.CREDENTIAL, encrypted = true) String credential,
+                                       @Param(Inputs.CommonInputs.PROXY_HOST) String proxyHost,
+                                       @Param(Inputs.CommonInputs.PROXY_PORT) String proxyPort,
+
```

Comment:
@Bonczidai  I know... just keep that line to make some delimitation between common inputs and specific inputs (to be more easier to follow)